### ISSUE TITLE
Background composition

### ISSUE THREAD

**Issue by gdjmck** [tsing90/pytorch_semantic_human_matting#8]
Hi, I wonder when generating training data, you composite the foreground image with a bunch of background images, it is for training purpose or to enlarge your training dataset?

The paper did not explain how the composition loss works and the former implementation of @lizhengwei1992 just uses the alpha to mask on the origin image, no background image is used.

**Comment by gdjmck** [comment]
Is the composition loss the main difference of pre-train m_net procedure between your implementation and @lizhengwei1992 's ? I didn't get acceptable result from pre-training m_net using @lizhengwei1992 's code.

**Comment by gdjmck** [comment]
Oh, sorry to bother you, I examine the paper and find out what I didn't understand.
